yt/data_objects/static_output.py

```diff
@@ -821,6 +821,15 @@ def _get_field_info(self, ftype, fname=None):
         # storing this condition before altering it
         guessing_type = ftype == "unknown"
         if guessing_type:
+            if fname in self.field_info._ambiguous_field_names:
+                msg = (
+                    f"The requested field name '{fname}' "
+                    "is ambiguous and corresponds to any one of "
+                    f"the following field types\n {self.field_info._ambiguous_field_names[fname]}. "
+                    "Please specify the requested field as an explicit "
+                    "tuple (ftype, fname)."
```

Comment:
I think it would be beneficial to inform the user which field it defaulted to in this pass, by including something like: "Defaulting to ('ftype', 'fname')."

Reply:
Should be good now!